master/internal/db/postgres.go

```diff
@@ -89,6 +89,19 @@ func Bun() *bun.DB {
 	return theOneBun
 }
 
+// Can't use tablesExist because it is a different database type.
+func databaseIsFresh() (bool, error) {
+	fresh, err := Bun().NewSelect().Table("pg_tables").
+		Where("schemaname = ?", "public").
+		Where("tablename = ?", "gopg_migrations").
```

Comment:
Is there a reason why this is being done dynamically? Do we plan to parameterize this query?

Reply:
switched to just using `tablename = "gopg..."`